### DISCUSSION TITLE
Docker or docker compose?

### DISCUSSION THREAD

**Discussion by simonedelmann** [Budibase/budibase#7466] [Help] [answered]
Hello there! I just started playing around with Budibase, replacing an old custom CRUD app with it.

When thinking about hosting, I wonder if there are any advantages or disadvantages using the all-in-one docker image vs. docker-compose? (Aside from better scalability with docker-compose) 

Thank you for your help!

**Comment by melohagan** [comment, marked as answer]
Hey @simonedelmann 

The docker single image should be smaller in size and may perform a little better. 
Docker compose on the other hand makes handling configuration easier. 

I found this article that may be of interest: https://cloudinfrastructureservices.co.uk/dockerfile-vs-docker-compose-whats-the-difference/
